docs/src/index.md

```diff
@@ -352,7 +352,7 @@ end
 ```
 ```jldoctest index
 #### Find dfoo/dx via rrules
-#### First the forward pass, accumulating rules
+#### First the forward pass, gathering up the pullbacks
```

Comment:
Perhaps `building`, `generating`, or `constructing` makes more sense here? `gathering` sounds to me like it means more collecting something that already exists.

Reply:
To my mind they already exist in the abstract. I think i like `gathering` better than `building`, `generating`, or `constructing `.
Thing is that we are doing two things.
We are constructing the pullbacks, and we are putting them onto the tape. Only the tape is implied here.

Reply:
Possibly this is a bit late, but I would prefer `computing the pullbacks` -- in general you have to do some work to get the pullback at a point.